contracts/neutron_interchain_txs/src/contract.rs

```diff
@@ -187,6 +193,28 @@ fn msg_with_sudo_callback<C: Into<CosmosMsg<T>>, T>(
     Ok(SubMsg::reply_on_success(msg, SUDO_PAYLOAD_REPLY_ID))
 }
 
+fn execute_set_fees(
+    deps: DepsMut,
+    denom: String,
+    _recv_fee: u128,
+    ack_fee: u128,
+    timeout_fee: u128,
+) -> StdResult<Response<NeutronMsg>> {
+    let fees = IbcFee {
+        recv_fee: vec![],
```

Comment:
Why don't you use `_recv_fee` here? :thinking:

Reply:
seems copy-paste issue